app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -108,6 +108,7 @@ public enum SettingsEnum {
     SB_SHOW_BROWSER_BUTTON("sb-browser-button", false, SharedPrefHelper.SharedPrefNames.SPONSOR_BLOCK, ReturnType.BOOLEAN),
     SB_API_URL("sb-api-url", "https://sponsor.ajay.app/api/", SharedPrefHelper.SharedPrefNames.SPONSOR_BLOCK, ReturnType.STRING);
 
+    public static boolean SB_SHORTS_ENABLED = false;
```

Comment:
Make sure to also adjust this in the [related patch](https://github.com/revanced/revanced-patches/pull/439).

```suggestion
    public static boolean shorts_playing = false;
```